landing.js

```diff
@@ -1,36 +1,59 @@
-$(function () {
-  initAutoComplete($("#autocomplete"), $("#goingnext"));
-  initAutoComplete($("#autocomplete_2"), $("#goingnext-2"));
-  initAutoComplete($("#autocomplete_3"), $("#goingnext-3"));
-  storeDataForProductPitchABTesting();
-});
-
 // used by the marketing team for analytics and AB testing, to track which page,
```

Comment:
The 4 files above aren't used?

Reply:
Yup, I couldn't find them anywhere in webflow. I'll have a deeper dive before merging, but looks like 🚮